src/components/overview/ImageGallery/DefaultView.jsx

```diff
@@ -2,33 +2,36 @@ import React, { useState, useEffect } from 'react';
 
 const DefaultView = ({state, updateState, updateMainImage, handleToExpand, handleRightArrow, handleLeftArrow}) => {
 
+
   return (
     <>
-      {(state !== undefined) && (
-        <div className='defaultView'>
-          <div className='styleImages'>
-            {state.selectedStyleDefaultImages.map((element, idx) => { //render all style photos
-              return <img
-                id={idx}
-                key={idx}
-                className='eachStyleImage'
-                onClick={updateMainImage}
-                src={element.url}/>;
-            })}
-          </div>
-          <div className='mainImage' id='mainImage'>
-            <img onClick={handleLeftArrow} className='btnFloating' id='left' src ='https://img.icons8.com/ios-filled/50/000000/long-arrow-left.png'></img>
-            {(state.selectedStyleDefaultImages[0] !== undefined && state.mainImage === '') ?
-              <div id='wrap'>
-                <img id ='main' src ={state.selectedStyleDefaultImages[0].url}/>
-              </div> :
-              <div onClick={handleToExpand} id='wrap'>
-                <img id ='main' src ={state.mainImage}/>
-              </div>
-            }
-            <img onClick={handleRightArrow} className='btnFloating' id='right' src ='https://img.icons8.com/ios-filled/50/000000/long-arrow-right.png'></img>
+      {state !== undefined && (
+        (state.currentStyle.photos !== undefined) && (
```

Comment:
I haven't seen this syntax curious to know how to interpret this!

Reply:
I actually just left comment at your PR about this lol. 

I did this because I wanted to render only after state.currentStyle.photos is updated from the parent component so that I can use map function, otherwise it gives me error saying that I cannot use map function because state.currentStyle.photos is undefined.